### PR TITLE
Create terraform.yml

### PR DESCRIPTION
**Pull request by Ninjaneer32**

### CODE REVIEW

File: .github/workflows/codeql.yml
Comment (stweil) [inline .github/workflows/codeql.yml:1]: Why do you add this file?

**Comment by stweil** [comment]
Please give more information why you made this pull request? How should it improve Tesseract?

File: SECURITY.md
[the remark this answers is not in the document]
Comment (Ninjaneer32) [reply SECURITY.md:1]: No

File: .github/workflows/codeql.yml
Comment (Ninjaneer32) [reply .github/workflows/codeql.yml:1]: Can

File: SECURITY.md
[the remark this answers is not in the document]
Comment (Ninjaneer32) [reply SECURITY.md:1]: 🤣

File: .github/workflows/codeql.yml
Comment (Ninjaneer32) [reply .github/workflows/codeql.yml:1]: KYS, fag

**Pull request closed without merging**